sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -21,6 +21,10 @@ namespace Azure.Messaging.WebPubSub
     [CodeGenSuppress("SendToGroupAsync", typeof(string),  typeof(RequestContent), typeof(IEnumerable<string>), typeof(RequestContext))]
     [CodeGenSuppress("SendToUser", typeof(string), typeof(RequestContent), typeof(RequestContext))]
     [CodeGenSuppress("SendToUserAsync", typeof(string), typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("AddUserToGroup", typeof(string), typeof(string), typeof(RequestContext))]
```

Comment:
These methods are published, so we need to preserve the signature

Reply:
Does CodeGenSuppress mean the method won't be generated, and so we author it in the custom layer?

Reply:
Yes, that is the only way of customization available right now.

Reply:
In the partial class, not in a separate layer though.